go/mysql/binlog.go

```diff
@@ -74,3 +75,23 @@ func (this *BinlogCoordinates) SmallerThanOrEquals(other *BinlogCoordinates) boo
 	}
 	return this.LogFile == other.LogFile && this.LogPos == other.LogPos
 }
+
+// IsLogPosOverflowBeyond4Bytes returns true if the coordinate endpos is overflow beyond 4 bytes.
```

Comment:
@shaohk last request, can we add a reference to why a value over 4 bytes is an overflow?

Reply:
OK, I have added. Please take a look again, Thks.